inst/rmd/qc_report.rmd

```diff
@@ -20,7 +20,27 @@ output:
 library(SingleCellExperiment)
 ```
 
-# `r params$sample` Experiment Information
+# Processing Information for `r params$sample`
+
+```{r echo=FALSE}
+# extract sce metadata containing processing information as table
+processing_info <- sce@metadata %>%
+  as.data.frame() %>%
```

Comment:
This works for the current structure of the metadata, but I am a bit worried about it, as it is entirely possible that we could end up with a metadata element that is not a single value. We also might have some metadata values we don't want to display.